Try redesigning the style of authoring-html

xfq opened the pull request:
Try redesigning the style of Internationalization techniques: Authoring web pages. Let me know what you think!

Current page: https://www.w3.org/International/techniques/authoring-html

[Preview](https://deploy-preview-665--i18n-drafts.netlify.app/techniques/authoring-html.en.html)

r12a commented:
It's certainly colourful.  There are however a number of things i think we need to consider changing. Perhaps a topic for our next 1:1.  Note also the comments on a previous commit about line lengths, that were shortened by the move to a new base style sheet.

Also, note that the headings overlap the expanded TOC - another result i think of the style sheet change.

xfq commented:
> the headings overlap the expanded TOC

I fixed a few issues, but not this one. I couldn't find the source of this issue, and I didn't modify any z-index related style...

xfq commented:
> > the headings overlap the expanded TOC
> 
> I fixed a few issues, but not this one. I couldn't find the source of this issue, and I didn't modify any z-index related style...

I just found that even if I deleted all the code I added, the problem still occurs:

<img width="996" alt="Screenshot 2025-06-03 at 15 08 57" src="https://github.com/user-attachments/assets/96e458eb-4baf-4b09-8e73-8b4699bea708" />

------

I will take a closer look to find out what caused it.

xfq commented:
Should be fixed now. Please take a look.

xfq commented:
@r12a After I discussed with you, I further modified the style, please take a look.

r12a commented:
There's still a problem with overlapping content when the browser window is narrow:

<img width="928" alt="Screenshot 2025-06-06 at 06 04 39" src="https://github.com/user-attachments/assets/85236ddc-fedf-443f-a52d-7384b20a9d7d" />

I think the width needs to be applied to the column, perhaps, rather than the menu?  (Btw, fwiw, i thought 15rem was probably wide enough when i did some testing — that also helps for narrow windows.)

xfq commented:
Merging.
